addon/components/nrg-loading-indicator.hbs

```diff
@@ -1,8 +1,8 @@
 <div
   class="ui loader
     {{if (eq @loading false) "disabled" "active"}}
-    {{unless (eq @centered false) "centered"}}
-    {{unless (eq @inline false) "inline"}}
+    {{if @centered "centered"}}
+    {{if @inline "inline"}}
```

Comment:
This may cause unintended side effects.
Previously, if `@centered` or `@inline` was `undefined` it would evaluate to true.
We should change these to `this.`, and use the `args !== false` method to default to true.